### ISSUE TITLE
Auto scroll chat to bottom when vs code start

### ISSUE THREAD

**Issue by ilan4ever** [continuedev/continue#3249]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

When start vscode and go to continue chat, user need to scroll down all the way to bottom to see the last conversation with llm.

### Solution

Make auto scroll to bottom when vs code start and user load extension.

**Comment by RomneyDa** [comment]
Yeah note that there's a commented-out useEffect in the chat page that used to do this
from here https://github.com/continuedev/continue/pull/2848

But the wait a bit and then scroll to bottom approach is pretty hacky and fragile

`flex-col-reverse` on `session.history` is the perfect solution but would be a big change and make backward compatibility with older sessions a real issue (I suppose we could check sessions on load and invert them if e.g. first message is user) but there's a lot of index -1 type of logic that would be sensitive

the challenge is detecting when the chat content has loaded so that the scroll height is valid

A solid but imperfect solution might be to
- store a map of index -> loaded on redux
- detect when styled markdown previews have loaded and have them dispatch updateIndexLoaded true
- when all have loaded (num loaded = history length), scroll to bottom, only once per session change

**Comment by RomneyDa** [comment]
@tomasz-stefaniak fixed with #3527
